assets/src/apps/scheduler/PageScheduleLine.tsx

```diff
@@ -6,21 +6,29 @@ import { ScheduleHeader } from './ScheduleHeader';
 import { DayGeometry } from './date-utils';
 import { getSelectedId } from './schedule-selectors';
 // import { SchedulePlaceholder } from './SchedulePlaceholder';
-import { HierarchyItem, moveScheduleItem, selectItem, unlockScheduleItem } from './scheduler-slice';
+import { HierarchyItem, moveScheduleItem, selectItem } from './scheduler-slice';
 
 interface ScheduleLineProps {
   item: HierarchyItem;
+  index: number;
   indent: number;
+  rowColor: string;
   dayGeometry: DayGeometry;
 }
 
-export const PageScheduleLine: React.FC<ScheduleLineProps> = ({ item, indent, dayGeometry }) => {
+export const PageScheduleLine: React.FC<ScheduleLineProps> = ({
+  item,
+  index,
+  indent,
+  rowColor,
+  dayGeometry,
+}) => {
   const dispatch = useDispatch();
   const isSelected = useSelector(getSelectedId) === item.id;
 
-  const onUnlock = useCallback(() => {
-    dispatch(unlockScheduleItem({ itemId: item.id }));
-  }, [dispatch, item.id]);
+  // const onUnlock = useCallback(() => {
```

Comment:
If this commented code is no longer used, should we just remove it?

Reply:
Done